src/lava/lib/optimization/solvers/generic/sconfig.py

```diff
@@ -1 +1,5 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
 num_in_ports = 1
```

Comment:
Are we using this to know how many concurrent replicas we are running? Would this prevent launching multiple OptimizationSolvers in parallel (in the same virtualenv)?

Reply:
Good catch, this is obsolete now, have removed the file ;)